cmd/create/oidcconfig/cmd.go

```diff
@@ -230,7 +230,7 @@ func run(cmd *cobra.Command, argv []string) {
 			}
 			roleName, _ := aws.GetResourceIdFromARN(args.installerRoleArn)
 			if roleName != "" {
-				if !output.HasFlag() && r.Reporter.IsTerminal() {
+				if !output.HasFlag() && r.Reporter.IsTerminal() && mode == aws.ModeAuto {
```

Comment:
The ticket says the installer role won't be used for the manual mode.
I wonder if all the other steps afterward: `ARNValidator`, `CheckRoleExists`, etc are applicable to the manual flow?

Reply:
@oriAdler Im wondering the same, do you know someone I can talk to?

Reply:
@gdbranco in the auto mode flow, should we skip lines `233-260`?

Reply:
We already skip it, as roleName would be empty in this case

What happens here is that the customer supplied the role arn even though it wasn't needed. We should probably fail first and say the role is not required for manual mode at the start.

But for this specific issue I  think it is enough to just not show the message